steps/scan-with-appsweep/1.0.0/step.yml

```diff
@@ -0,0 +1,60 @@
+title: AppSweep Mobile Security Testing
+summary: Scan your Android app using Guardsquare's mobile application security testing
+  tool AppSweep
+description: "The build of your app is automatically uploaded to [AppSweep](https://appsweep.guardsquare.com)
+  for scanning. \n\nAppSweep is Guardsquare's free tool enabling developers to put
+  mobile security at the forefront of app development. AppSweep helps you identify
+  and fix security issues in your code and dependencies with actionable recommendations
+  and insights that will help you build more secure mobile apps.\n"
+website: https://www.guardsquare.com/appsweep-mobile-application-security-testing
+source_code_url: https://github.com/Guardsquare/bitrise-step-upload-to-appsweep
```

Comment:
This link is 404 for me. Also, it doesn't match the `source` field below.

Reply:
This indeed was the wrong link.

Does it need to be the same as the Website? The website would explain what AppSweep is, the source code is for this step.